akka-actor-typed-tests/src/test/scala/docs/akka/typed/FSMDocSpec.scala

```diff
@@ -43,7 +43,7 @@ object FSMDocSpec {
     // initial state
     def apply(): Behavior[Event] = idle(Uninitialized)
 
-    private def idle(data: Data): Behavior[Event] = Behaviors.receiveMessage[Event] { message: Event =>
+    private def idle(data: Data): Behavior[Event] = Behaviors.receiveMessage[Event] { (message: Event) =>
```

Comment:
this can probably be:
```
Behaviors.receiveMessage[Event] { message =>
```

Reply:
Done. Used the suggested code in new commits